api/package/get-supporting-companies.js

```diff
@@ -0,0 +1,10 @@
+const S = require('fluent-schema')
+const { CompaniesSupportingPackage } = require('../../types/Package')
+const { Base200WithData } = require('../../common/reply')
+
+module.exports = {
+  querystring: S.object().prop('id', S.string()).required(),
```

Comment:
Can you use the `PackageId` from the `types/Package` file? I think there's one in there for package ids.